requirements_dev.txt

```diff
@@ -6,7 +6,7 @@ pytest-notebook
 nbsphinx
 nbval>=0.9.6
 nbconvert
-sphinx>=1.7
+Sphinx==2.4.4
```

Comment:
Interesting, I just did this in another bird as well. Should we perhaps loosen the requirement found in `wps-tools`?

Reply:
Ya that's a good idea